api-reference/_hidden/dxDataGridColumn/calculateGroupValue.md

```diff
@@ -13,8 +13,40 @@ The current row's data.
 The calculated value used to create and sort groups.
 
 ---
+This property accepts the name of the data source field that provides these values...
 
-The **DataGrid** groups its rows by the grouping column. If you want to use custom grouping, change values of this column in the **calculateGroupValue** function. As a result, group cells' headers will display the changed values that may include extra information. To remove it, specify a custom template in the [groupCellTemplate](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#groupCellTemplate) property ([groupCellRender](/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/columns/#groupCellRender) in React). Refer to this <a href="https://github.com/DevExpress-Examples/DataGrid---How-to-apply-custom-sorting-to-a-grouped-column" target="_blank">Github repository</a> for an example. 
+    {
+        dataField: "Country",
+        groupIndex: 0,
+        calculateGroupValue: "groupDataField"
+    }
+
+... or a function that calculates them:
+
+    // groups records by date
+    calculateGroupValue: function(rowData){
```

Comment:
Both the code snippets should be divided by platforms. And since we are editing the calculateSortValue topic, it would be great to add Vue and React samples to it as well.